gtsam/sfm/ShonanAveraging.h

```diff
@@ -151,6 +164,15 @@ class GTSAM_EXPORT ShonanAveraging {
     return measurements_[k];
   }
 
+  /// wrap factors with robust Huber loss
+  static Measurements makeNoiseModelRobust(Measurements measurements){
+	  Measurements robustMeasurements = measurements;
```

Comment:
I’m not in favor of this imperative scheme, but FWIW you are passing by value so the copy was already made :-)